src/java.base/share/classes/jdk/internal/loader/BootLoader.java

```diff
@@ -83,6 +72,8 @@ private BootLoader() { }
     private static final NativeLibraries NATIVE_LIBS
         = NativeLibraries.jniNativeLibraries(null);
 
+    private static final JavaLangAccess JLA = SharedSecrets.getJavaLangAccess();
```

Comment:
It be nicer if you could move this to the top to be consistent with the other files.